content/guides/integrations/memcached/Python.md

```diff
@@ -0,0 +1,248 @@
+---
+title: "PyMemache"
+date: 2018-08-24T01:00:08-07:00
+draft: false
+aliases: [/integrations/memcached/python]
+---
+
+![memcached logo](/img/memcached-python.png)
+
+- [Introduction](#introduction)
+- [Features](#features)
+- [Using it](#using-it)
+- [Enabling OpenCensus](#enabling-opencensus)
+    - [Enabling tracing](#enabling-tracing)
+- [End to end example](#end-to-end-example)
+- [Examining your traces](#enabling-your-traces)
+- [References](#references)
+
+## Introduction
+[Memcached](https://memcached.org) is one of the most used server caching and scaling technologies.
+
+It was created by [Brad Fitzpatrick](https://en.wikipedia.org/wiki/Brad_Fitzpatrick) in 2003 as a
+solution to scale his social media product [Live Journal](https://www.livejournal.com/)
+
+Pinterest's Python [PyMemcache](https://github.com/pinterest/pymemcache) client has been forked and [trace instrumented with OpenCensus](https://github.com/opencensus-integrations/pymemcache)
```

Comment:
nit: trace instrumented with OpenCensus -> instrumented with OpenCensus tracing

Reply:
Roger that thanks!